--- neverball-1.5.4/Makefile.linking	2011-12-07 11:11:55.656533331 +0100
+++ neverball-1.5.4/Makefile	2011-12-07 11:21:35.671468461 +0100
@@ -138,7 +138,7 @@ ifdef DARWIN
 endif
 
 ALL_LIBS := $(SDL_LIBS) $(BASE_LIBS) $(TILT_LIBS) $(INTL_LIBS) -lSDL_ttf \
-    -lvorbisfile $(OGL_LIBS)
+    -lvorbisfile $(OGL_LIBS) -lX11
 
 #------------------------------------------------------------------------------
 
@@ -318,7 +318,7 @@ $(PUTT_TARG) : $(PUTT_OBJS)
 	$(CC) $(ALL_CFLAGS) -o $(PUTT_TARG) $(PUTT_OBJS) $(LDFLAGS) $(ALL_LIBS)
 
 $(MAPC_TARG) : $(MAPC_OBJS)
-	$(CC) $(ALL_CFLAGS) -o $(MAPC_TARG) $(MAPC_OBJS) $(LDFLAGS) $(BASE_LIBS)
+	$(CC) $(ALL_CFLAGS) -o $(MAPC_TARG) $(MAPC_OBJS) $(LDFLAGS) $(BASE_LIBS) -lm
 
 # Work around some extremely helpful sdl-config scripts.
 
